new/LIT7222QeneAgbertihu.xml

```diff
@@ -0,0 +1,74 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT7222QeneAgbertihu" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">አመ፡ አግብርቲሁ፡ መሀከ፡ እግዚአብሔር፡</title>
+                <title xml:lang="gez" corresp="#t1" type="normalized">ʾAma ʾagbǝrtihu mahaka ʾƎgziʾabǝḥer (Qǝne possibly of wāzemā-type)</title>
```

Comment:
I would leave out "possibly" here. The grade of certainty can be better discuss un <abstract/>